README.md

```diff
@@ -0,0 +1,91 @@
+# Renuo Thumbs Proxy
+
+Renuo thumbs proxy is an app to sign images so they are processed by thumbor. You can restrict the source of the images by setting
+the BACKEND_IMAGES_PATH (e.g. ao3ief5j.cloudfront.com), so only images from this domain / path are processed by thumbor.
+
+## Deployment
+
+The app is ready to run on heroku. Create a new heroku app, set the env variables, and push the code to Heroku.
+
+### Performance / Requests per Second
+
+With the default heroku host (1 dyno for the proxy, 1 dyno for thumbor), about 20 rps are possible.
+
+## Installation and Usage
+
+* Install pyenv
+* cp local_server.example.sh local_server.sh
+* Adjust config for local_server.sh
+
+```sh
+pyenv install
+pip install -r requirements.txt
+THUMBOR_SECURITY_KEY=UJwHAZLsRejTyLI88lAriHL7xAXa6q0umiwwpPcP \
+ BACKEND_ASSET_PATH=ao3ief5j.cloudfront.com \
+ THUMBOR_PATH=thumbor.example.com \
+ DEBUG=True \
+ python server.py
+```
+
+* Create a backend and upload images (e.g. S3)
+* Make the images accessible (e.g. Cloudfront)
+** From this step, you will get the BACKEND_ASSET_PATH, e.g. ao3ief5j.cloudfront.com
+* Generate a THUMBOR_SECURITY_KEY, configure thumbor
+* Transform images calling the right url (/thumb/<path:config>/u/<path:uri>)
+** Example: /thumb/200x300/smart/u/images/logo.png
```

Comment:
On github nested lists need to be indented with two spaces: https://help.github.com/articles/markdown-basics/#nested-lists